ivy/functional/backends/paddle/experimental/elementwise.py

```diff
@@ -132,7 +132,7 @@ def gcd(
     *,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    return paddle.gcd(x1, x2)
```

Comment:
add this before returning the function `x1, x2 = promote_types_of_inputs(x1, x2)`

Reply:
the suggested solution is better than this one, because it handles also the dtypes of both inputs, while the to_tensor function determines the dtype regardless of the other input